pkg/templates/data.go

```diff
@@ -191,11 +191,16 @@ func GetDeployIgnitionTemplateData(targetDevice, postScript string, sparseClone,
 	}
 }
 
-func GetRegistryEnv(registryData, registryUpgrade string) string {
+func GetRegistryEnv(registryImage, registryData, registryUpgrade string, useOcpRegistry bool) string {
+	ocpRegistryFlag := "false"
+	if useOcpRegistry {
```

Comment:
why not just using the 'useOcpRegistry' flag? i.e. instead of adding also ocpRegistryFlag